CHANGELOG.md

```diff
@@ -1,16 +1,13 @@
-# Changelog
+## [](https://github.com/paralus/paralus/compare/v0.2.3...v) (2023-05-02)
```

Comment:
can you check the comparison version .. `compare/v0.2.3...v)`

Reply:
@akshay196 so is there a way we can add a link to compare current version with prior given only date is displayed ?

Reply:
we can live with it as of now.